.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["react", "es2015", "stage-0"],
+  "presets": [
+    "react",
+    ["latest", {"es2015": {"modules": false}}],
+    "stage-0"
+  ],
```

Comment:
[`latest` is actually deprecated](http://babeljs.io/docs/plugins/preset-latest/) in favour of using [`env`](http://babeljs.io/docs/plugins/preset-env/), which, if we’re changing it anyway, I think we should use instead!

It lets us frame the compilation in regards to browser compatibility rather than “versions” of JavaScript!